docs/schemas_and_objects.md

```diff
@@ -0,0 +1,74 @@
+# Unigraph's Schemas and Objects
+
+As part of our [Goal](./goal.md), Unigraph has a user-centric, free principle about schemas and objects in a flexible graph database. 
```

Comment:
just noting I think there is no `Goal.md` at the moment

Reply:
True, I'm thinking writing such a higher-level documentation might be better if we both chip in, maybe we can do it over a future synchronized call?